### ISSUE TITLE
It should be possible to use arrays in html data-ot-* attributes.

### ISSUE THREAD

**Issue by Burtsev-Alexey** [enyo/opentip#58]
data-ot-hide-triggers='[ "closeButton", "trigger" ]'   - I want to hide tooltip on close or trigger element.

**Comment by Vinalynn** [comment]
i have the same problem

**Comment by enyo** [comment]
Yeah I'm sorry... Arrays aren't working as html attributes right now. I'll implement that in the next version.

For now, you'll have to create the Opentips programmatically if you want to do so.
